zerver/openapi/zulip.yaml

```diff
@@ -10233,6 +10276,35 @@ paths:
                               read messages.
 
                               **Changes**: New in Zulip 5.0 (feature level 105).
+                      user_topics:
+                        type: array
+                        description: |
+                          Present if `user_topic` is present in `fetch_event_types`.
+                        items:
+                          type: object
+                          description: |
+                            Object describing the relationship between a user-topic pair.
```

Comment:
This sentence is not being rendered in the documentation. The `realm_linkifiers` object array also returned by this endpoint would be a good example to follow for formatting this description in the documentation. Again note that you'll want to include a **Changes** note with the feature level here.

Reply:
I followed the `muted_users` array documentation to write this. Just noticed that a similar sentence there, describing the object, doesn't get rendered as well.